src/pallets_sphinx_themes/__init__.py

```diff
@@ -5,7 +5,11 @@
 import textwrap
 from collections import namedtuple
 
-import importlib_metadata
+try:
```

Comment:
Sorry, should have caught this the first time. Move this below the normal imports so reorder-python-imports continues to work.

Reply:
Bah I should've caught it too, sorry. Fixed and rebased again.